save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/LnkUserProjectController.kt

```diff
@@ -0,0 +1,59 @@
+/**
+ * Controller for processing links between users and their roles:
+ * 1) to put new roles of users
+ * 2) to get users and their roles by project
+ */
+
+package org.cqfn.save.backend.controllers
+
+import org.cqfn.save.backend.security.ProjectPermissionEvaluator
+import org.cqfn.save.backend.service.LnkUserProjectService
+import org.cqfn.save.backend.service.ProjectService
+import org.cqfn.save.domain.Role
+import org.cqfn.save.info.UserInfo
+import org.cqfn.save.permission.Permission
+
+import org.slf4j.LoggerFactory
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api")
+/**
```

Comment:
BTW, I was sure that diktat would place docs before annotations. We should check if we have this rule

Reply:
I also think there should be something, but I couldn't find it yet. Still placed kdoc before annotations.